### ISSUE TITLE
[core] "Unknown option: '-min'"  but is referenced in documentation

### ISSUE THREAD

**Issue by dschach** [pmd/pmd#4454]
**Affects PMD Version:** 

7.0.0-rc1

**Description:**

The `-min` parameter is no longer allowed. I cannot use the priority (1-5) for rules to exclude some from being checked.

`--minimum priority` is a Low/Medium/High value, so that can not be the solution:
```
Rule priority threshold; rules with lower priority than configured here won't be used. Valid values (case insensitive): High, Medium High, Medium, Medium Low, Low
```

Substituting --minimum-priority gives a long error message basically saying that it is not a number, but a string.

[-min is referenced in documentation](https://docs.pmd-code.org/latest/pmd_userdocs_configuring_rules.html#message-and-priority-overriding)

```
Run ~/pmd/bin/pmd check -d force-app -d sample-handler --rulesets pmd/ruleset.xml --format csv --no-cache -min 2 --verbose -r pmd/pmd.csv
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by org.codehaus.groovy.reflection.CachedClass (file:/home/runner/pmd/lib/groovy-2.[4](https://github.com/dschach/salesforce-trigger-framework/actions/runs/4622271015/jobs/8174740004#step:4:5).21.jar) to method java.lang.Object.finalize()
WARNING: Please consider reporting this to the maintainers of org.codehaus.groovy.reflection.CachedClass
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
Unknown option: '-min'
Possible solutions: --minimum-priority
Error: Process completed with exit code 2.
```

**Code Sample demonstrating the issue:**

```bash
~/pmd/bin/pmd check --dir force-app --dir sample-handler --rulesets pmd/ruleset.xml --format csv --no-cache -min 2 --verbose -r pmd/pmd.csv
```

**Steps to reproduce:**

Please provide detailed steps for how we can reproduce the bug.

Run pmd with `-min` parameter and 

**Running PMD through:** *GitHub Action*: https://github.com/dschach/salesforce-trigger-framework/blob/main/.github/workflows/ci.yml

**Comment by adangel** [comment]
Thanks for reporting!
That's indeed wrong in the documentation.
In your case, the old `-min 2` needs to be replaced with `--minimum-priority "Medium Low"` to achieve the same behavior.
